Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -86,22 +86,22 @@ - (id)init {
     static NSString* browserUserAgentString = nil;
 
     void (^setUpBrowserUserAgent)() = ^() {
-        browserUserAgentString =
-            [[[UIWebView alloc]
-              initWithFrame:CGRectZero]
-                stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
+		if (!browserUserAgentString) {
```

Comment:
This is out of scope for this PR, but I would propose checking this value every 30 days similar to what we do with strong matching instead with every request and cache it in the pref helper. AFAIU, this value will not change unless the user updates their OS. Allocating a `WebView` for every single request is a bit expensive.